examples/meta/src/regression/least_angle_regression.sg

```diff
@@ -14,11 +14,11 @@ Labels labels_test = labels(f_labels_test)
 PruneVarSubMean SubMean()
 NormOne Normalize()
 SubMean.fit(features_train)
-SubMean.apply_to_feature_matrix(features_train)
-SubMean.apply_to_feature_matrix(features_test)
+features_train = wrap(SubMean.apply(features_train))
```

Comment:
wrap? This should not be in the .sg file, it is automatically generated, see https://github.com/shogun-toolbox/shogun/blob/develop/examples/meta/generator/targets/cpp.json#L14

Reply:
@karlnapf yeah i realized this is wrong. we should update generator to automatically wrap the raw pointer here

Reply:
It does that!

Reply:
@karlnapf unfortunately it doesn't :(
generated cpp doesn't compile. `features_train` is Some, which is implicitly cast to raw pointer, and `apply` always return raw pointer here